src/store/getters/state-getters.ts

```diff
@@ -512,8 +490,8 @@ export const isFilingValid = (state: StateIF): boolean => {
   return false // should never happen
 }
 
-/** Whether any correction sections are in editing mode. */
-export const isEditing = (state: StateIF): boolean => {
+/** Whether the subject correction filing has any sections in editing mode. */
+export const isCorrectionEditing = (state: StateIF): boolean => {
```

Comment:
Renamed to emphasize this is for corrections only. It is the old architecture for testing whether any components are in editing mode.

(In the new architecture, I believe components are invalid if they are in editing mode, which is caught by the new "is valid" code.)